ts/src/renderer/phaser/scenes/GameScene.ts

```diff
@@ -203,7 +205,15 @@ class GameScene extends PhaserScene {
 			map.width * map.tileWidth / 2 * scaleFactor.x,
 			map.height * map.tileHeight / 2 * scaleFactor.y
 		);
-		camera.zoom = this.scale.width / 800;
+	}
+
+	private setZoomSize (height: number): void {
+		this.zoomSize = height * 2.15;
```

Comment:
I think we should leave a comment about this value, 2.15 being arbitrary

Reply:
yes, revert previous comment